static/js/hotkey.js

```diff
@@ -348,6 +348,11 @@ export function process_enter_key(e) {
         return true;
     }
 
+    if (popovers.message_info_popped()) {
+        popovers.user_info_popover_for_message_handle_keyboard("enter");
+        return true;
+    }
+
```

Comment:
This seems like what's missing, but it does strike me have like 8 semi-duplicate blocks here, one for each popover.  I'm not sure there's a good refactor to clean that up.  We also have 2 copies of this, one just for `"enter"` and the other for all the other keys.

Maybe the right answer as a step for now is to extract the `menu_dropdown_hotkeys.has` block as a function, and just call that from here (passing `enter` as the parameter)?  

That'd at least remove this copy and make it a lot harder to mess this up when adding a new popover or whatever.